src/coreclr/jit/compiler.h

```diff
@@ -6035,6 +6035,9 @@ class Compiler
     // Mark a loop as removed.
     void optMarkLoopRemoved(unsigned loopNum);
 
+    // Condition uses of ValueNumStore
+    ValueNum optConservativeVN(GenTree* tree);
```

Comment:
Sorry, I realize that this was the name I gave you, but if you are going to need to rerun CI anyway, can you please rename this to `optConservativeNormalVN`?

Reply:
Is there a better comment?

Reply:
Maybe something like "During global assertion prop, returns the conservative normal VN for a tree; otherwise returns NoVN`?